src/applications/gi/components/vet-tec/EybVetTecCalculatorForm.jsx

```diff
@@ -0,0 +1,191 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+
+import recordEvent from 'platform/monitoring/record-event';
+import { formatCurrency } from '../../utils/helpers';
+import { ariaLabels } from '../../constants';
+import Dropdown from '../Dropdown';
+import { calculatorInputChange } from '../../actions';
+import { connect } from 'react-redux';
+import RadioButtons from '../RadioButtons';
+
+class EybVetTecCalculatorForm extends React.Component {
+  constructor(props) {
+    super(props);
+    const selectedProgram =
+      this.props.selectedProgram !== ''
+        ? this.props.selectedProgram
+        : this.props.preSelectedProgram;
+
+    this.state = {
+      tuitionFees: 0,
+      scholarships: 0,
+      programName: selectedProgram,
+    };
+  }
+
+  setProgramFields = programName => {
+    if (programName) {
+      const program = this.props.institution.programs.find(
+        p => p.description.toLowerCase() === programName.toLowerCase(),
+      );
+      if (program) {
+        const field = 'EYBVetTecProgram';
+        const value = {
+          vetTecTuitionFees: this.state.tuitionFees,
+          vetTecProgramName: this.state.programName,
+          vetTecScholarships: this.state.scholarships,
+          vetTecProgramFacilityCode: this.props.institution.facilityCode,
+        };
+        this.props.calculatorInputChange({ field, value });
+      }
+    }
+  };
+
+  handleApprovedProgramsChange = event => {
+    const vetTecProgramName = event.target.value;
+    const program = this.props.institution.programs.find(
+      p => p.description.toLowerCase() === vetTecProgramName.toLowerCase(),
+    );
+
+    this.setState({
+      programName: vetTecProgramName,
+      tuitionFees: program.tuitionAmount,
+    });
+
+    this.props.calculatorInputChange({ vetTecProgramName });
+  };
+
+  trackChange = (fieldName, event) => {
+    const value = +event.target.value.replace(/[^0-9.]+/g, '');
+
+    recordEvent({
+      event: 'gibct-form-change',
+      'gibct-form-field': fieldName,
+      'gibct-form-value': value,
+    });
+  };
+
+  calculateBenefitsOnClick = event => {
+    event.preventDefault();
+    this.setProgramFields(this.state.programName);
+    document.getElementById('estimated-benefits-header').focus();
```

Comment:
Would it be better to use React useRef instead of jQuery to set focus? https://reactjs.org/docs/refs-and-the-dom.html

Reply:
There's a platform util `focusElement` you should use

Reply:
That isn't JQuery.

Reply:
You are correct. Use the utility Devin mentioned.

Reply:
That's a great method. It looks like it sets a tab index for you so you don't have to worry about creating eslint errors.